main.tf

```diff
@@ -48,19 +48,11 @@ resource "tfe_workspace" "default" {
 resource "tfe_notification_configuration" "default" {
   count                 = var.slack_notification_url != null ? 1 : 0
   name                  = tfe_workspace.default.name
-  enabled               = true
   destination_type      = "slack"
+  enabled               = true
```

Comment:
This should set based on the length of `var. slack_notification_triggers`; the notification configuration shouldn't exist if there aren't any triggers defined.